Vic2ToHoI4/Data_Files/converterIdeas.txt

```diff
@@ -753,6 +753,30 @@
 				}
 			}
 		}
+
+		reclaim_cores_idea = {			
+			
+			allowed = {
+				always = no
+			}
+
+			cancel = {
+				NOT = {
+					has_dynamic_modifier = { modifier = revanchism }
+					has_dynamic_modifier = { modifier = revanchism_fasc }
```

Comment:
What happens with this in a world with no fascism?

Reply:
[Run ideas through the Updater](https://github.com/ParadoxGameConverters/Vic2ToHoI4/pull/568/commits/8217dfb0471f76fa8322d3846ae1cef3becbbd8f)